app/demo-loop/simple-cms.org.json

```diff
@@ -0,0 +1,224 @@
+{
+"_format": "SCALE ORANGE",
+"_version": 0,
+"materials": {
+"cell_to_mat": [
+-1,
+5,
+0,
+1,
+2,
+3,
+4,
+5
+],
+"names": [
+"si",
+"pb",
+"c",
+"ti",
+"fe",
+"galactic"
+]
+},
+"universes": [
+{
+"_type": "simple unit",
+"bbox": [
+[
+-1000.0,
+-1000.0,
+-2000.0
+],
+[
+1000.0,
+1000.0,
+2000.0
+]
+],
+"cell_names": [
+"[EXTERIOR]",
+"vacuum_tube",
+"si_tracker",
+"em_calorimeter",
+"had_calorimeter",
+"sc_solenoid",
+"fe_muon_chambers",
+"world"
+],
+"cells": [
+{
+"faces": [
+0,
+1,
+2,
+3,
+4,
+5
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 & 3 ~ & 4 & 5 ~ & ~",
+"num_intersections": 6,
+"zorder": 1
+},
+{
+"faces": [
+6,
+7,
+8
+],
+"logic": "0 ~ 1 & 2 ~ &",
+"num_intersections": 4,
+"zorder": 1
+},
+{
+"faces": [
+6,
+7,
+8,
+9
+],
+"flags": 1,
+"logic": "0 ~ 1 & 2 ~ & ~ 1 2 ~ & 3 ~ & &",
+"num_intersections": 6,
+"zorder": 1
+},
+{
+"faces": [
+7,
+8,
+9,
+10
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 ~ & ~ 0 1 ~ & 3 ~ & &",
+"num_intersections": 6,
+"zorder": 1
+},
+{
+"faces": [
+7,
+8,
+10,
+11
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 ~ & ~ 0 1 ~ & 3 ~ & &",
+"num_intersections": 6,
+"zorder": 1
+},
+{
+"faces": [
+7,
+8,
+11,
+12
+],
+"flags": 1,
+"logic": "0 1 ~ & 2 ~ & ~ 0 1 ~ & 3 ~ & &",
+"num_intersections": 6,
```

Comment:
Just curious -- in general, how is the number of intersections determined?

Reply:
It's the sum of the number of intersection per surfaces over all surfaces. So the inner and outer cylinder each have two, and the Z caps each have one, for a total of 6.

Reply:
Well, that seems obvious in retrospect now :blush: